sites/all/themes/orwell/sass/component/_search.scss

```diff
@@ -0,0 +1,135 @@
+@import 'bootstrap/bootstrap';
+
+$color__search-form: #d8d8d8;
+$item-height: 6.875rem;
+$icon-color: #9c9c9c;
+
+.pane-search-form {
+  height: $item-height;
+  margin-left: auto;
+
+  @include grid-media($medium) {
+    height: auto;
+    width: 100%;
+  }
+}
+
+.search-form {
+  border-top: 1px solid $color__search-form;
+  font-family: $font-regular;
+
+  &__icon__wrapper {
+    align-items: center;
+    cursor: pointer;
+    display: flex;
+    height: $item-height;
+    justify-content: center;
+
+    &:before {
+      @include icon;
+      color: $icon-color;
+      content: $icon__search;
+      font-size: 2.5rem;
+      position: absolute;
+      z-index: 99999;
+
+      @include grid-media($medium) {
+        color: $color__white;
+        position: relative;
+        font-size: 1.5rem;
+        top: 5px;
+      }
+    }
+
+    &.open {
+      &:before {
+        content: '\ea0f';
+        font-size: 1.5rem;
+
+        @include grid-media($medium) {
+          font-size: 1rem;
+          top: 8px;
+        }
+      }
+
+      @include grid-media($medium) {
+        position: relative;
+        left: calc(50% - 1rem);
+        width: 2rem;
+        height: 2rem;
+      }
+    }
+
+    @include grid-media($medium) {
+      height: 2rem;
+      background: $red;
+      display: block;
+      text-align: center;
+    }
+  }
+
+  &__icon {
+    width: 4rem;
+
+  }
+
+  &__content {
+    display: none;
+    width: 100%;
+    height: $item-height;
+    position: absolute;
+    max-width: 100%;
+    top: 0;
+    left: 0;
+    z-index: 4;
+    background: white;
+
+    @include grid-media($medium) {
+      position: relative;
+      width: 1200px;
+      margin: 0 auto;
+    }
+  }
+
+  input#edit-search-block-form--2 {
```

Comment:
Isn't there a better way to target this? The ID is generated by Drupal, and might change. the `--2` thing is because Drupal rendered the form twice (whyfor), and will go away if we figure out how to loose the double rendering of the form.

Reply:
agree